testng-asserts/src/main/java/org/testng/Assert.java

```diff
@@ -1750,7 +1765,8 @@ public static void assertEquals(Iterable<?> actual, Iterable<?> expected, String
    * @param expected the expected value
    * @param message the assertion error message
    */
-  public static void assertEquals(Object[] actual, Object[] expected, String message) {
+  public static void assertEquals(
+      @Nullable Object[] actual, @Nullable Object[] expected, @Nullable String message) {
```

Comment:
I believe it should be `@Nullable Object @Nullable [] actual`: nullable array of nullable elements